airbyte-config/config-models/src/main/resources/types/OperatorWebhookInput.yaml

```diff
@@ -0,0 +1,26 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/OperatorWebhookInput.yaml
+title: OperatorWebhookInput
+description: Execution input for a webhook operation
+type: object
+required:
+  - name
+  - executionUrl
+properties:
+  name:
+    description: The user-friendly name of the operation, used for logging.
+    type: string
+  executionUrl:
+    description: URL to invoke the webhook via POST.
+    type: string
+  executionBody:
+    description: Message body to be POSTed.
+    type: string
+  webhookConfigId:
+    type: string
+    format: uuid
```

Comment:
can we add a description explaining that this is the id used to index into the `workspaceWebhookConfigs` object?

Reply:
it's can be slightly confusing otherwise